docs/basics/read_nwb.ipynb

```diff
@@ -71,13 +71,25 @@
   },
```

Comment:
<p>should remove trailing spaces</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>